### PR TITLE
Improve wording of OpenGL support on desktop platforms.

### PR DESCRIPTION
**Pull request by danila-zol**

### CODE REVIEW

File: contributing/development/core_and_modules/internal_rendering_architecture.rst [hunk 1]
```diff
@@ -228,8 +228,8 @@ support Vulkan. OpenGL 3.3 Core Profile is used on desktop platforms to run this
 driver, as most graphics drivers on desktop don't support OpenGL ES.
 WebGL 2.0 is used for web exports.
 
-It is possible to use the use of OpenGL ES 3.0 directly on desktop platforms
-using the ``--rendering-driver opengl3_es`` command line argument, although this
+It is possible to use OpenGL ES 3.0 directly on desktop platforms
```
Comment (bruvzg) [inline contributing/development/core_and_modules/internal_rendering_architecture.rst:231]: I think it's only supposed on Linux, not on all desktop platforms. Windows and macOS can use GLES over Angle, but not directly.

Comment (Calinou) [reply contributing/development/core_and_modules/internal_rendering_architecture.rst:231]: Indeed, Linux drivers have support for it but Windows/macOS drivers don't.

**Review by Calinou — APPROVED** [review]
Thanks!
Thanks! Congratulations for your first merged pull request :tada:

**Comment by mhilbrunner** [comment]
Cherrypicked to 4.3 in https://github.com/godotengine/godot-docs/pull/10347.

**Pull request merged**